services/venice-controller/src/main/java/com/linkedin/venice/controller/server/StoresRoutes.java

```diff
@@ -827,7 +835,7 @@ public void internalHandle(Request request, MultiStoreTopicsResponse veniceRespo
           });
           veniceResponse.setTopics(deletableTopicsList);
         } catch (Exception e) {
-          veniceResponse.setError("Failed to list deletable store topics. Message: " + e.getMessage());
+          veniceResponse.setError("Failed to list deletable store topics. Message: " + e.getMessage(), e);
```

Comment:
nit: Just wondering if we use ", e" pattern, do we need to log error message in the error log?

Reply:
Here, `setError("...", e)` will only extract the `ErrorType` from the exception. It will be sent in the response to the client. The log is for the server-side logging. https://github.com/linkedin/venice/blob/main/internal/venice-client-common/src/main/java/com/linkedin/venice/controllerapi/ControllerResponse.java#L68

Reply:
I just re-read the code. It seems the message will get appended twice. Will update it